testing/test_scripts/check_install.sh

```diff
@@ -0,0 +1,19 @@
+###############################################################################
+
+# This script tests the if a perticular tool is installed or not
+
+#Usage:
+# $ sh check_install.sh <tool_name>
+
+################################################################################
+
+#!/bin/bash
+
+if [ `which $1 2>&1 >/dev/null | grep -c no` -eq 1 ]
```

Comment:
yes, thats a simpler solution. I have sent another pull request with this change . Please check.